tf2onnx/tfonnx.py

```diff
@@ -422,22 +422,15 @@ def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
     # kernel must to be transposed
     if with_kernel:
         parent = node.inputs[1]
-        if node.inputs[1].is_const():
-            # kernel is const - transpose the const
-            if not parent.data_format:
-                val = parent.get_tensor_value(as_list=False)
-                val = val.transpose(HWCN_TO_NCHW)
-                parent.set_tensor_value(val)
-        else:
-            # kernel comes from op, insert transpose op
-            input_name = node.input[1]
-            transpose = ctx.insert_new_node_on_input(node, "Transpose", input_name)
-            transpose.set_attr("perm", HWCN_TO_NCHW)
-            transpose.inserted_nchw = True
-            ctx.copy_shape(input_name, transpose.output[0])
-            new_shape = spatial_map(ctx.get_shape(input_name), HWCN_TO_NCHW)
-            ctx.set_shape(transpose.output[0], new_shape)
-            nodes.append(transpose)
+        # kernel comes from op, insert transpose op
```

Comment:
plz copy & paste the commit description here as well.

Reply:
I thought the code reader may not need to know there is a bug here before.